lib/rouster.rb

```diff
@@ -188,7 +188,29 @@ def initialize(opts = nil)
 
         raise ArgumentError.new('AWS passthrough requires valid :sshkey specification, should be path to private half') unless File.file?(@passthrough[:key])
         @sshkey = @passthrough[:key]
+      elsif @passthrough[:type].eql?(:openstack)
+        @logger.debug(sprintf('instantiating an %s passthrough worker', @passthrough[:type]))
+        @sshkey = @passthrough[:key]
+
+        ostack_defaults = {
+          :ssh_port              => 22,
+          :user                  => 'vagrant',
+        }
+	@passthrough = ostack_defaults.merge(@passthrough)
+
+	[:openstack_auth_url, :openstack_username, :openstack_tenant, :openstack_api_key, :connection_options,
```

Comment:
not sure that `:connection_options` should be required, as the examples show us passing an empty hash